src/components/LoadingSpinner/_loading-spinner.scss

```diff
@@ -0,0 +1,8 @@
+@import "../../scss/variables";
+
+.loading-spinner {
```

Comment:
Similar loading alignment was used in a few places. I think this will need a followup PR to make some tweaks after https://github.com/canonical/juju-dashboard/pull/1547 lands.